### DISCUSSION TITLE
A Quick Setup Guide for LEMP Stack, Laravel and Supervisord with custom AWS EC2 AMI

### DISCUSSION THREAD

**Discussion by giscus** [ducxinh/nix-tech-blog-discussion#1] [General]
# A Quick Setup Guide for LEMP Stack, Laravel and Supervisord with custom AWS EC2 AMI

A Quick Setup Guide for LEMP Stack and Laravel with Custom AWS EC2 AMI: Ubuntu 22.04, Laravel 10, PHP 8.2, Mysql 8.0, Supervisord

http://localhost:3000/blog/infra-and-devops/quick-setup-guild-lemp-stack-laravel-on-ec2

**Comment by ducxinh** [comment]
shared

**Comment by phamtu613** [comment]
Vip luôn idol ơi 😘
